ROADMAP.md

```diff
@@ -1,62 +1,191 @@
+
+Each Appendix header is linked to a GitHub Epic. Latest information on progress can be found in the Epics and child issues.
+
+## 🛣️ Milestones
+
+### 2022
+
+#### Early Q4 (October)
+- [A.1 User Configured Interop Tests & Dashboard](#1-user-configured-interop-tests--dashboard)
+
+#### Mid Q4 (November)
+- [A.2 Interop tests for all existing/developing features](#2-interop-tests-for-all-existingdeveloping-features)
+- [A.3 Rock solid libp2p releases](#3-rock-solid-libp2p-releases)
+- [C.1 Benchmarking using nix-builders](#1-benchmarking-using-nix-builders)
+
+#### End of Q4 (December)
+- [A.4 Canonical Interop Tests & Dashboard](#4-canonical-interop-tests--dashboard)
+
+### 2023
+
+#### Early Q1 (January)
+
+- add here
+
+#### Mid Q1 (February)
+
+- add here
+
+#### End of Q1 (March)
+- [C.2 Benchmarking using remote runners](#2-benchmarking-using-remote-runners)
+
+### Up Next
+
+## 📖 Appendix
+
+**Projects are listed in descending priority.**
+
+### A. [Multi-dimensional Testing/Interop Visibility](https://github.com/libp2p/test-plans/issues/53)
+**Why:** libp2p supports a variety of protocols (multiple transports, muxers, & security protocols) across implementations in different languages. Until we actually test them together, we can't guarantee implementation interoperability. We need to ensure and demonstrate that: interoperable features work with each other as expected and we don't introduce degradations that break interoperability in new releases.
+
+**Goal:** libp2p implementers run tests across permutations of libp2p implementations, versions, and supported protocols. Implementers and users can reference a single website with a dashboard to view the Pass/Fail/Implemented/Not Implemented results of multi-dimensional tests.
+
+**How:**
+#### 1. [User configured interop tests & dashboard](https://github.com/libp2p/test-plans/issues/55)
+Enable test-plans maintainers to define a configuration (of libp2p impls, versions, protocols, expected RTT), run Testground tests per configuration, and retrieve resulting data in a standard format.
+The test case results can be formatted as a "dashboard" (simple Markdown table of Pass/Fail results.)
+
+#### 2. Interop test plans for all existing/developing features
+<!-- TODO: Create issue -->
+Using tooling from A.1, all features of go-libp2p and rust-libp2p that should be interoperable are tested (i.e. transports (TCP, QUIC, WebRTC, WebTransport), multiplexers (mplex, yamux), secure channels (TLS, Noise), etc.) across versions.
+
+Features currently in development across implementations (like WebRTC in go-libp2p and rust-libp2, or QUIC & TLS in rust-libp2p) are not merged without at least manually running these test suites.
+
+Test suites are run in libp2p/test-plans CI.
+
+**Note:** node.js-libp2p is not listed here since it does not have support in Testground yet.
+
+#### 3. Rock solid libp2p releases
```

Comment:
This seems like a _really easy_ change. Once we have the test (2.), this is literally just a small change to our release GitHub Actions workflow.

I'm wondering if this warrants a separate section here.

Reply:
I added it to the milestone above.